library/xml-model/src/test/java/org/cru/godtools/xml/model/tract/CardTest.kt

```diff
@@ -25,9 +30,10 @@ class CardTest {
         val listenerEvents = Event.Id.parse(TOOL_CODE, "listener1 listener2")
         val dismissListenerEvents = Event.Id.parse(TOOL_CODE, "dismiss-listener1 dismiss-listener2")
 
-        val card = parseCardXml("card.xml")
+        val card = parseCardXml("card.xml", )
```

Comment:
you shouldn't have the trailing comma here, guessing that's one of the lint failures